<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientChatWidget</class>
 <widget class="QWidget" name="ClientChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="lstMessage"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="wdgSend">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>消息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lnMessage"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblOpenFolder">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;#&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;open&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
